arena.js

```diff
@@ -0,0 +1,145 @@
+/* arena.js */
+
+class Arena {
+	constructor(name) {
+		this.name = capitalize(name);
+		this.gladiators = [];
+	}
+	addGladiator(gladiator) {
+		console.log(`Welcome to the arena, ${gladiator.name}.`);
+		this.gladiators.push(gladiator);
```

Comment:
shouldn't we check to see if there are not already two gladiators in the arena before we push...